pom.xml

```diff
@@ -296,7 +315,7 @@
                     <inherited>false</inherited>
                     <configuration>
                         <includes>**/pyproject.toml</includes>
-                        <excludes>**/target/**</excludes>
+                        <excludes>**/target/**,**/.venv/**</excludes>
```

Comment:
I: Now that venvs are stored local to Habushu modules, add them to ignore so we don't try to commit the pyproject.toml files inside.